projects/uitdatabank/docs/search-api/pagination.md

```diff
@@ -0,0 +1,73 @@
+# Pagination
+
+Every search endpoint supports pagination using two url parameters:
+-   `start`: the number of results to skip (`int`, defaults to 0)
+-   `limit`: the number of results to return in a single page (`int`, defaults to 30)
+
+Pagination will only work for a limited result set:
+-   The maximum value for for `start` is set to 10.000
+-   The maximum value for `limit` is set to 2.000
+
+<!-- theme: info -->
+
+> The response also includes the limit included in the request, as `itemsPerPage`. This is because the API might decide to return fewer results if the specified limit is too high. Because of this, you should never assume that the specified limit will also be respected and you should always check `itemsPerPage`.
```

Comment:
I also read this in the old documentation, but it's actually not true (anymore). We now just return an error if a limit is given that is too high. So the `itemsPerPage` is always the same as the given `limit` (or the default value if none is given)